src/modules/Elsa.EntityFrameworkCore.MySql/Migrations/Alterations/20231015122151_Initial.cs

```diff
@@ -8,18 +9,23 @@ namespace Elsa.EntityFrameworkCore.MySql.Migrations.Alterations
     /// <inheritdoc />
     public partial class Initial : Migration
     {
+        private readonly IElsaDbContextSchema _schema;
+        public Initial(IElsaDbContextSchema schema)
+        {
+            _schema = schema ?? throw new ArgumentNullException(nameof(schema));
+        }
```

Comment:
Is this generated, or do we need to manually do this after generating migrations?

Reply:
For now , we have to add this manually .

Maybe we can generate this automatically , I know we can hook some element in the generation, but afaik empty constructor is the basic usage in EF so I don't know if we can hook this element in the generation, I've to check this  and I'll propose a new pr if any

Reply:
OK, thanks for clarifying.